rdkit-postgresql/meta.yaml

```diff
@@ -4,22 +4,28 @@ package:
 
 source:
   git_url: https://github.com/rdkit/rdkit.git
-  git_tag: Release_2016_09_2
+  git_rev: Release_2017_03
   patches:
-    - Makefile.patch
+    - cmakelists.txt.patch
     - adapter.cpp.patch
 
 build:
   number: 1
 
 requirements:
   build:
+    - cmake
     - python
-    - rdkit >=2016.09
+    - msys2-conda-epoch >=20160418 [win]
+    - m2-msys2-runtime [win]
+    - m2-libiconv [win]
+    - m2-libintl [win]
+    - m2-diffutils [win]
+    - m2-patch [win]
     - postgresql >=9.4,<9.5
+    - rdkit >=2016.09
```

Comment:
is the build-time dependency from `rdkit >=2016.09` a leftover? (the *95 version of the cartridge depends on `rdkit >=2017.03`)

also, it seems rdkit is no longer a runtime dependency (this also applies to the *95 version), is it intended?

Reply:
1. Yes, it is a leftover
2. No, it is not intended
Thanks for checking those - you have much more experience than I do with the cartridge, so this is very welcome and helpful!